docker/pypi/wmagent/run.sh

```diff
@@ -1,5 +1,29 @@
 #!/bin/bash
 
+_find_child_pss() {
+    # An auxiliary function to find all processes forked from the shell parrent
+    # of the current process
+    cat  /proc/$PPID/task/*/children
+}
+
+_service_gracefull_exit() {
+    # An auxiliary function to handle the WMAgent service graceful exit
+    manage stop-agent
+    wait
+
+    for proc in _find_child_pss
+    do
```

Comment:
@anpicci Can you add something to debug when we enter this loop when testing?

e.g.

```
echo "SIGKILL: $proc
```

Reply:
I just did in my latest commit